src/pages/developing/react-tutorial/wrapping-up.mdx

```diff
@@ -18,7 +18,7 @@ Badge.
 <AnchorLinks>
 
 <AnchorLink>Application form</AnchorLink>
-<AnchorLink>Badge information</AnchorLink>
+<AnchorLink>Badge process</AnchorLink>
```

Comment:
This link is broken because the heading is still "Badge information" below?

Reply:
resolved. :)